pkg/client/apiutil/dynamicrestmapper.go

```diff
@@ -94,25 +95,38 @@ func WithCustomMapper(newMapper func() (meta.RESTMapper, error)) DynamicRESTMapp
 	}
 }
 
+// WithDiscoveryFilter sets a groupVersion filter for the discovery client
+// that allows to limit the relevant groups for the mapper
+func WithDiscoveryFilter(filter func(groupVersion string) bool) DynamicRESTMapperOption {
```

Comment:
This needs better godoc -- does `true` mean `keep` or `discard`?

Reply:
done in godoc